plutus-conformance/test-cases/uplc/evaluation/untyped-plutus-core-tests/iteAtHigherKindFullyApplied/iteAtHigherKindFullyApplied.uplc

```diff
@@ -0,0 +1,15 @@
+(program
```

Comment:
I'm pretty sure some of these are only interesting for type reasons. e.g. this one says something about "higher kinds", which obviously aren't there in the _untyped_ tests. So I think we may want to delete some of these. Testing that if-then-else works at various types is good, though. Perhaps we just need to rename some of them if they're not redundant?

Reply:
Maybe I should just call them `test1`, `test2`, ...: that'd solve that problem.  It's getting to the point where it's difficult to work out which tests are redundant/duplicated because there are so many now.  I suppose I could defend that on the basis that it's better to have too many tests rather than too few.

Reply:
I agree that it's going to be annoying and tedious to sort them out, but I think we should do it anyway, otherwise it will only get worse over time.

Reply:
Hm yeah I think it's best to clean this out too, I'm not sure how though!